.github/workflows/nightly-test-nvidia.yml

```diff
@@ -446,147 +393,30 @@ jobs:
         run: |
           IS_BLACKWELL=1 bash scripts/ci/ci_install_dependency.sh
 
-      - name: Run Mistral-Large-3 nightly performance test
-        timeout-minutes: 180
+      - name: Run B200-specific model tests
+        if: always()
+        timeout-minutes: 200
         env:
           TRACE_BASE_URL: https://raw.githubusercontent.com/sglang-bot/sglang-ci-data/main/traces/${{ github.run_id }}
           PERFETTO_RELAY_URL: ${{ vars.PERFETTO_RELAY_URL }}
           GPU_CONFIG: "8-gpu-b200"
           SGLANG_ENABLE_JIT_DEEPGEMM: "0"
         run: |
           rm -rf test/performance_profiles_mistral_large3/
-          rm -rf test/performance_profiles_mistral_large3_eagle/
           cd test
-          IS_BLACKWELL=1 python3 nightly/test_mistral_large3_perf.py
-
-      - name: Publish Mistral-Large-3 traces to storage repo
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_PAT_FOR_NIGHTLY_CI_DATA }}
-          GITHUB_RUN_ID: ${{ github.run_id }}
-          GITHUB_RUN_NUMBER: ${{ github.run_number }}
-        run: |
-          python3 scripts/ci/publish_traces.py --traces-dir test/performance_profiles_mistral_large3
-          python3 scripts/ci/publish_traces.py --traces-dir test/performance_profiles_mistral_large3_eagle
+          IS_BLACKWELL=1 python3 run_suite.py --hw cuda --suite nightly-8-gpu-b200-basic --nightly --timeout-per-file=12000 --continue-on-error
```

Comment:
Let's only have common and we can skip tests in the file if needed. If we have more divergence in the future we can create new files